pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -247,13 +248,13 @@ func (o *OvirtMapper) MapDataVolumes() (map[string]cdiv1.DataVolume, error) {
 		}
 		quantity, _ := resource.ParseQuantity(diskSizeConverted)
 
-		dvs[attachID] = cdiv1.DataVolume{
+		dvs[dvName] = cdiv1.DataVolume{
 			TypeMeta: metav1.TypeMeta{
 				APIVersion: cdiAPIVersion,
 				Kind:       dataVolumeKind,
 			},
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      attachID,
+				Name:      dvName,
```

Comment:
Why not to use `GenerateName` here? We could use myattchID- and k8s would make sure it is unique. In `dvs` slice we could use myattchID as a key and find dv using `strings.HasPrefix`

Reply:
Thinking about it more we can use target vm name as a prefix.